docs/services-developer-guide.md

```diff
@@ -161,10 +163,9 @@ They can be added by hand, but new services are encouraged to use [`angryjet`]
 to generate them automatically using `go generate` per the `angryjet`
 documentation.
 
-Note that in many cases a suitable provider, resource claim, and portable
-resource class will already exist. Frequently adding support for a new managed
-service requires only the definition of a new managed resource and non-portable
-resource class.
+Note that in many cases a suitable provider and resource claim will already
+exist. Frequently adding support for a new managed service requires only the
+definition of a new managed resource and resource class.
```

Comment:
Probably fine because we say "Frequently", but I don't know if it would be useful to point out network resources and other types that do not have a class.

Reply:
True, though that could change if https://github.com/crossplaneio/crossplane/issues/968 becomes a reality. Since this is arguably good enough I'm going to take the path of least resistance and leave it as is for now.